resource-service/common/git.go

```diff
@@ -164,6 +169,14 @@ func (g Git) rewriteDefaultBranch(path string, env envconfig.EnvConfig) error {
 
 func (g Git) init(gitContext common_models.GitContext, projectPath string) (*git.Repository, error) {
 	init, err := g.git.PlainInit(projectPath, false)
+	if err != nil {
+		if errors.Is(err, git.ErrRepositoryAlreadyExists) {
+			init, err = g.git.PlainOpen(projectPath)
+		} else {
+			return nil, err
+		}
+	}
+
 	if err != nil {
 		return nil, err
 	}
```

Comment:
I would move this lines at line 174. Here they are confusing since there is 2 if err != nil in a row